src/main/java/io/confluent/idesidecar/restapi/connections/DirectConnectionState.java

```diff
@@ -356,4 +356,4 @@ public static Throwable unwrap(Throwable t) {
     }
     return t;
   }
-}
+}
```

Comment:
This diff is extraneous and may be reverted. (A new line must be added back to the end of the file)